tests/bin/prune-search-index_test.py

```diff
@@ -1,18 +1,30 @@
 from tests.functional_test_base import FunctionalTestBase
-from shelf.bulk_update.utils import run_search_prune
+from shelf.bucket_update.utils import prune_search_index
```

Comment:
I did validate it is running. Pylint is also complaining that it's an invalid module name though... Fixed. cff638f